vendor/bat-native-ads/src/bat/ads/internal/platform/platform_helper.cc

```diff
@@ -15,7 +15,6 @@ PlatformHelper::~PlatformHelper() = default;
 
 void PlatformHelper::set_for_testing(
     PlatformHelper* platform_helper) {
-  DCHECK(platform_helper);
```

Comment:
@tmancey why are we removing DCHECK here?

Reply:
Some tests require this to be reset to a `nullptr`